stdlib/LinearAlgebra/test/dense.jl

```diff
@@ -84,6 +84,15 @@ bimg  = randn(n,2)/2
         end
     end # for eltyb
 
+@testset "Test diagm for vectors" begin
+    @test diagm(zeros(50)) == diagm(0 => zeros(50))
+    @test diagm(ones(50)) == diagm(0 => ones(50))
+    for i = 1:50
```

Comment:
I don't really think this needs to be done 50 times. Once is enough to test that it's working.

Reply:
Ok, so I scrape it?

Reply:
Yes, just remove the loop and keep the inner part.